multiversx_sdk_rust_contract_builder/source_code.py

```diff
@@ -50,7 +50,7 @@ def get_source_code_files(
     files_related_to_contract = set(file.path for file in source_code_files)
 
     if include_unrelated_to_contract:
-        all_files = get_all_files(contract_folder, _is_source_code_file)
+        all_files = get_all_files(project_folder, _is_source_code_file)
```

Comment:
This was a bug. But it didn't manifest, since `include_unrelated_to_contract` was always `False` (with the exception of integration tests).